src/DatePicker/__stories__/DatePicker.stories.js

```diff
@@ -23,33 +19,33 @@ storiesOf('DatePicker', module)
       content: DatePickerReadme,
     },
   })
-  .add('default', () => {
+  .add('uncontrolled state', () => {
     const localeValue = select(
       'Locale',
       {
         English: 'en',
         French: 'fr',
       },
       'English',
-      'Locale',
+      'Props',
     );
 
-    const withMinDate = boolean('With minimum date', false, 'Bounds');
-    const minDateValue = date('Minimum date', today, 'Bounds');
-    const withMaxDate = boolean('With maximum date', false, 'Bounds');
-    const maxDateValue = date('Maximum date', today, 'Bounds');
+    const withMinDate = boolean('With minimum date', false, 'Props');
+    const minDateValue = date('Minimum date', today, 'Props');
+    const withMaxDate = boolean('With maximum date', false, 'Props');
+    const maxDateValue = date('Maximum date', today, 'Props');
 
     const numberOfMonthsToDisplay = number(
-      'Number of month to display',
+      'Number of months to display',
       1,
       { range: true, min: 1, max: 2, step: 1 },
-      'Bounds',
+      'Props',
     );
 
-    const isRange = boolean('Range date picker', false, 'Type');
+    const isRange = boolean('Range date picker', false, 'Props');
 
     return (
-      <div style={{ background: 'white', padding: '16px', borderRadius: '4px' }}>
+      <Wrapper style={{ background: 'white', padding: '16px', borderRadius: '4px' }}>
```

Comment:
Why do you apply style twice here ?

Reply:
I didn't, just didn't pay attention to existing style :) will update